modules/sr/robot/ruggeduino_devices.py

```diff
@@ -4,6 +4,10 @@
 
 
 class DistanceSensor:
+    """
+    A standard Webots distance sensor. Unfortunately there is a 30cm range limit within Webots.
+    We convert the distance to metres as to match the standard SR API.
```

Comment:
Does the SR API have ultrasound sensors built-in? I'm not aware that it does (though j5 and the SourceBots APIs may do).